.codex/skills/material3-expressive-android/references/color-system.md

```
@@ -0,0 +1,40 @@
+# Color system
+
+## Sources
+
+- https://developer.android.com/develop/ui/compose/designsystems/material3
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)